packages/wrangler/src/__tests__/output.test.ts

```diff
@@ -0,0 +1,229 @@
+			writeOutput({
+				type: "wrangler-session",
+				version: 1,
+				wrangler_version: "0.0.0.0",
+				command_line_args: ["--help"],
+			});
+			writeOutput({
+				type: "deployment",
+				version: 1,
+				worker_id: "Worker",
+				deployment_id: "1234",
+			});
+
+			const outputFile = readFileSync(WRANGLER_OUTPUT_FILE_PATH, "utf8");
+			expect(outputFile).toContainEntries([
+				{
+					type: "wrangler-session",
+					version: 1,
+					wrangler_version: "0.0.0.0",
+					command_line_args: ["--help"],
+				},
+				{
+					type: "deployment",
+					version: 1,
+					worker_id: "Worker",
+					deployment_id: "1234",
+				},
+			]);
+		} finally {
+			process.env = originalProcessEnv;
+		}
+	});
+
+	it("should write to a random file in WRANGLER_OUTPUT_FILE_DIRECTORY", () => {
+		try {
+			process.env = {
+				...processEnvNoVars,
+				WRANGLER_OUTPUT_FILE_DIRECTORY: "output",
+			};
+			writeOutput({
+				type: "wrangler-session",
+				version: 1,
+				wrangler_version: "0.0.0.0",
+				command_line_args: ["--help"],
+			});
+
+			const outputFilePaths = readdirSync("output");
+			expect(outputFilePaths.length).toEqual(1);
+			expect(outputFilePaths[0]).toMatch(/wrangler-output-.+\.json/);
+			const outputFile = readFileSync(
+				join("output", outputFilePaths[0]),
+				"utf8"
+			);
+			expect(outputFile).toContainEntries([
+				{
+					type: "wrangler-session",
+					version: 1,
+					wrangler_version: "0.0.0.0",
+					command_line_args: ["--help"],
+				},
+			]);
+		} finally {
+			process.env = originalProcessEnv;
+		}
+	});
+
+	it("should write multiple entries to the same random file in WRANGLER_OUTPUT_FILE_DIRECTORY", () => {
+		try {
+			process.env = {
+				...processEnvNoVars,
+				WRANGLER_OUTPUT_FILE_DIRECTORY: "output",
+			};
+			writeOutput({
+				type: "wrangler-session",
+				version: 1,
+				wrangler_version: "0.0.0.0",
+				command_line_args: ["--help"],
+			});
+			writeOutput({
+				type: "deployment",
+				version: 1,
+				worker_id: "Worker",
+				deployment_id: "1234",
+			});
+
+			const outputFilePaths = readdirSync("output");
+			expect(outputFilePaths.length).toEqual(1);
+			expect(outputFilePaths[0]).toMatch(/wrangler-output-.+\.json/);
+			const outputFile = readFileSync(
+				join("output", outputFilePaths[0]),
+				"utf8"
+			);
+			expect(outputFile).toContainEntries([
+				{
+					type: "wrangler-session",
+					version: 1,
+					wrangler_version: "0.0.0.0",
+					command_line_args: ["--help"],
+				},
+				{
+					type: "deployment",
+					version: 1,
+					worker_id: "Worker",
+					deployment_id: "1234",
+				},
+			]);
+		} finally {
+			process.env = originalProcessEnv;
+		}
+	});
+});
+
+expect.extend({
+	toContainEntries(received: string, expected: OutputEntry[]) {
+		const outputEntries = received
+			.trim()
+			.split("\n")
+			.map((line) => JSON.parse(line))
+			.map(({ timestamp, ...entry }) => {
+				expect(typeof timestamp).toBe("string");
+				console.log(entry);
```

Comment:
Has this been left intentionally?

Reply:
Oops!

Reply:
Removed: https://github.com/cloudflare/workers-sdk/pull/6383/commits/08d006463afca5c3d68ae93b76e0da3aba58375c